### PR TITLE
Mm/GitHub actions

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
Adding github actions to cover the following user-stories

1. install the dependencies with yarn and npm maybe we can choose
2. All the tests are running (npm test)
3. The linter is OK (npm lint)
4. The nextjs part and the server part are building
5. The security checker is OK (npm audit)
6. I can bundle the lib (npm npm export-build) covered by number 4

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/package.json [hunk 1]
```diff
@@ -38,7 +38,6 @@
     "atob": "^2.1.2",
     "babel-plugin-module-resolver": "^4.0.0",
     "babel-plugin-prismjs": "^2.0.1",
-    "base64-img": "^1.0.4",
```
Comment (Miniplop) [inline packages/slice-machine/package.json:37]: this package is not used anymore ?

File: packages/slice-machine/next.config.js [hunk 2]
```diff
@@ -6,15 +6,15 @@ const withCustomBabelConfigFile = require('next-plugin-custom-babel-config');
...
       // overridden by start script
       CWD: path.resolve("./tests/project"),
     },
     webpack: (config, { isServer }) => {
       if (!isServer) {
-        config.node = {
-          fs: "empty",
-        };
+        config.node = { fs: "empty" };
+        //config.resolve.fallback = { fs: false }
```
Comment (Miniplop) [inline packages/slice-machine/next.config.js]: maybe we can remove this comment no ?

**Review by Miniplop — COMMENTED** [review]
Nice changes, just to really undrstand why we need to update next, react and react-dom to make the CI works ?

**Comment by Miniplop** [comment]
Maybe we can add prettier as eslint plugin no ? To handle the code formatting ?

File: packages/slice-machine/package.json (hunk 1 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/package.json:37]: couldn't find it any where when running.
`grep -irn "base64-img" server src start helpers hooks lib templates components pages`

File: packages/slice-machine/next.config.js (hunk 2 above)
Comment (MarcMcIntosh) [reply packages/slice-machine/next.config.js]: yes, that was for using web-pack version 5, but I've set next to use 4 for now

**Review by Miniplop — APPROVED** [review]
OMG here we are !!!!

**Pull request merged**